psychopy/tests/test_monitors/test_monitors.py

```diff
@@ -0,0 +1,30 @@
+from psychopy.monitors.calibTools import Monitor
+import pytest
+
+
+@pytest.mark.monitors
+class TestMonitorCalibration(object):
+    def setup(self):
+        pass
+
+    def teardown(self):
+        pass
+
+    def test_reload_monitor(self):
+        self.mon = Monitor('test',
+                           width=40,
+                           distance=57,
+                           gamma=1.0,
+                           notes='Here are notes')
+        self.mon.saveMon()
+        self.mon2 = Monitor('test')
+        assert all([[self.mon.calibs[key][key2] is self.mon2.calibs[key][key2]
```

Comment:
Wow, this looks quite complex! Without looking at the code (I'm on my way to bed :)), wouldn't something like `assert mon.calibs == mon2.calibs` suffice?

Reply:
It would almost work, but there's a numpy array in the `Monitor` object that gums up the works (`gammaGrid`). I've expended the comparison to hopefully make it a bit clearer (and fixed it, the above code didn't work properly). Thank you for looking this over!